src/index.ts

```diff
@@ -1,152 +1,152 @@
-import net from 'node:net';
-import tls from 'node:tls';
-import { Duplex } from 'node:stream';
-import type { ReadableStream, WritableStream } from 'node:stream/web';
-import type { SocketAddress, SocketInfo, SocketOptions } from './types';
-import { isSocketAddress } from './is-socket-address';
-
-export function connect(
+import net from 'node:net'
```

Comment:
Insert blanket preference for trailing semi-colons here.

#TeamSemicolonFTW

Reply:
I switched to https://github.com/neostandard/neostandard , which does have a semi option. I'll enable it in a follow up 😉